core/common/libp2p/cbor_stream.cpp

```diff
@@ -40,8 +42,7 @@ namespace fc::common::libp2p {
             return cb(count.error());
           }
           self->buffer_.resize(self->size_ + count.value());
-          self->consume(gsl::make_span(self->buffer_).subspan(self->size_),
-                        std::move(cb));
+          self->consume(gsl::make_span(self->buffer_).subspan(self->size_), cb);
```

Comment:
why not move `cb`?

Reply:
`std::move of the const variable 'cb' has no effect`.
`cb` is captured by value and is `const`.

Reply:
ok.  
can remove `move`.  
or add `mutable` to lambda.